dmscripts/helpers/supplier_data_helpers.py

```diff
@@ -17,24 +19,26 @@ class SupplierFrameworkData(object):
 
     data = None
 
-    def __init__(self, client, target_framework_slug):
+    def __init__(self, client, target_framework_slug, supplier_ids=None):
```

Comment:
The `AppliedToFrameworkSupplierContextForNotify` subclass is missing this (or a `**kwargs`) - I'm getting a failure when I try doing a `--dry-run`.

Reply:
Fixed

Reply:
Also added some unit tests and functional tests to catch issues with this script in future